server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -537,7 +539,7 @@ public void testTableConstraintsWithOrderBy() {
     @Test
     public void testDefaultColumns() {
         execute("select * from information_schema.columns order by table_schema, table_name");
-        assertEquals(923, response.rowCount());
+        assertEquals(935, response.rowCount());
```

Comment:
Is this correct? to list the columns of pg_XXX tables in `information_schema`?

Reply:
Yes, `information_schema.columns` should contain columns of all tables, including system tables of `pg_catalog` or the `information_schema` schema. E.g. it also list columns of the table `information_schema.columns` itself.

Reply:
thx!